v2f/src/main/scala/org/broadinstitute/monster/etl/v2f/V2FExtractionsAndTransforms.scala

```diff
@@ -116,4 +148,27 @@ object V2FExtractionsAndTransforms {
     )
     ()
   }
+
+  /**
+    *  Merge the variant JSON Objects and then write the merged JSON to disk.
+    *
+    * @param variantJsonAndFilePaths a list of the collections of JSON Objects and associated file paths that will be merged and then saved as a JSON file
+    * @param outputDir the root outputs directory where the JSON file(s) will be saved
+    */
+  def mergeVariantJsonAndFilePathsAndWriteToDisk(
+    variantJsonAndFilePaths: List[SCollection[(String, JsonObject)]],
+    outputDir: String
+  ): Unit = {
+    SCollection
+      .unionAll(variantJsonAndFilePaths.map { collection =>
+        collection.map {
+          case (_, jsonObj) =>
+            jsonObj
+        }
+      })
+      .saveAsJsonFile(
```

Comment:
Two things here:
1. The call to `saveAsJsonFile` should happen in the main pipeline, not this method. That way everything in this file is about transforming data.
2. Where the `saveAsJsonFile` call is now, add a `distinct`. There are some duplicate records in the latest output.